merging with Alkalus 5.08

e99999 opened the pull request:
merging with Alkalus 5.08

e99999 committed:
merging with Alkalus 5.08

merging with Alkalus 5.08

e99999 committed:
fixing ignoring

fixing ignoring

e99999 committed:
fix ignores again

ignores

e99999 committed:
final stupid ignoring shit

sorry if this bugs anyone im stupid and cant do this right

diregarath commented:
@e99999 Looks like you were trying to remove the idea files and logs :)  👍

e99999 committed:
deleting log

log be gone

e99999 committed:
more ingore

more ignoring

e99999 committed:
removing useless folder

hopefully it will ignore when i re-add to mine

e99999 committed:
making grass greener

adding brighter color map to foliage, can be tweaked later.

leagris commented:
Please remove https://github.com/bear989/GT5u-Bears-Den/pull/3/commits/138985c8667ca7720d27ebcac81cd0bbf47cce1a as this belong to a texture/resource pack as end-user preference.

e99999 committed:
added cofh lib & changed grass

cofh added to dependancy library, grass is no longer universally greener. Grass stuff is subject to removal leagris : )

e99999 committed:
grass stuff removed

cause every got pissy : )

diregarath commented on .classpath:
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Does this need to be committed?  Seems like an IDE file.

diregarath commented on .gitignore:
```diff
@@ -1,73 +1,10 @@
+*.xml
+*.log
+*.iml
+*.log
```
Do we need *.log twice?

diregarath replied:
Any reason we are removing the comments that make it easier to understand the gitignore?

diregarath commented on .project:
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
IDE file?

diregarath commented on build.gradle:
```diff
@@ -28,9 +28,6 @@ version = "${config.gt.version}"
 group= "gregtech"
 archivesBaseName = "gregtech"
 
-sourceCompatibility = 1.8
```
Shouldn't we build against java 8?  why not?

diregarath commented on build.properties:
```diff
@@ -1,11 +1,11 @@
 minecraft.version=1.7.10
-forge.version=10.13.4.1614-1.7.10
```
I switched this to 1614 - I would like to keep it on the last version of 1.7.10 forge.

diregarath commented on compile_Gregtech.cmd:
```diff
@@ -0,0 +1,3 @@
+@echo on
```
What is this for?  gradlew build is easy enough to run.

diregarath commented on gradlew:
```diff
@@ -1,164 +0,0 @@
-#!/usr/bin/env bash
```
Why are we removing gradlew and adding the bat version? I often develop on os x and linux (and cygwin on windows) - this needs to be restored.  We should have both.

diregarath commented on .gitignore:
```diff
@@ -1,73 +1,6 @@
-### Windows ###
```
I liked how this was organized so we knew why specific things were being excluded.  We should bring it back if possible.

diregarath reviewed (changes requested):
Please don't merge this until at least gradlew is back, we are building java 1.8 (or do we have a reason to stay with java 7?), and we are using forge 10.13.4.1614-1.7.10.

Commit:
test commit

Commit:
remove project files, use java 8, etc.

Commit:
add targetCompatibility = 1.8
